tex/latex/pgf-pie/tikzlibrarypie.code.tex

```diff
@@ -441,9 +453,13 @@
       \pgfmathparse{sqrt(\pgfpie@p) * (\pgfpie@polarRadiusUnit)}
       \xdef\pgfpie@theradius{\pgfmathresult}
     }{%
-    % normal pie
-    \pgfmathaddtolength{\pgfpie@angleEnd}{\pgfpie@p}
-    }
+        % normal pie
+        \pgfpie@ifchangedirection{%
+            \pgfmathaddtolength{\pgfpie@angleEnd}{-\pgfpie@p}
+        }{%
+            \pgfmathaddtolength{\pgfpie@angleEnd}{\pgfpie@p}
+        }%
+    }%
```

Comment:
Please use two spaces for indentation and use parentheses in mathematical expression just to guard for the eventuality of someone somehow injecting an expression like `1+1` into `\pgfpie@p`.
```suggestion
      % normal pie
      \pgfpie@ifchangedirection{%
        \pgfmathaddtolength{\pgfpie@angleEnd}{-(\pgfpie@p)}
      }{%
        \pgfmathaddtolength{\pgfpie@angleEnd}{\pgfpie@p}
      }%
    }%
```